exercise/do-your-own-map.js

```diff
@@ -0,0 +1,73 @@
+/*
+## Do-Your-Own-Map
+
+### Instructions
+
+Create a `MapFunc` function that takes an array as first argument, a function as second,
```

Comment:
no caps in function names, also avoid saying it's a function, so the name would be `map`

By conventions, capitalized functions are constructors in javascript, like the actual `Map` which is a data-struct constructor, let's just call this one `map`, we could add `flatMap` too just for fun.

Reply:
noted for the no cap comment.
Added flatMap.